Source/WebCore/page/SecurityOrigin.cpp

```diff
@@ -596,6 +596,23 @@ Ref<SecurityOrigin> SecurityOrigin::create(const String& protocol, const String&
     return origin;
 }
 
+Ref<SecurityOrigin> SecurityOrigin::create(WebCore::SecurityOriginData&& data, const String& domain, const String& filePath, const Markable<OpaqueOriginIdentifier, OpaqueOriginIdentifier::MarkableTraits>& opaqueOriginIdentifier, bool universalAccess, bool domainWasSetInDOM, bool canLoadLocalResources, bool enforcesFilePathSeparation, bool needsStorageAccessFromFileURLsQuirk, std::optional<bool> isPotentiallyTrustworthy, bool isLocal)
```

Comment:
I'm curious why the SecurityOriginData is an r-value but domain and filePath aren't.  It think they should all be.